### DISCUSSION TITLE
LskyPro后台图片管理增加图片链接

### DISCUSSION THREAD

**Discussion by Matinal111** [lsky-org/lsky-pro#419] [想法]
LskyPro在前端页面上传图片能复制对应的链接，但在后台图片管理要查看某张图片的链接就需要点开图片然后在点下缩略图浏览器跳转到一个新标签页才能复制链接:zany_face:，后续更新能点开图片在属性显示对应的链接吗？

**Comment by 0xxb** [comment]
有时间会对这方面功能进行改进的。
